### PR TITLE
Add Autoconf 2.13

### PR DESCRIPTION
**Pull request by satmandu**
This is needed for some packages.

Works properly:
- [x] x86_64
- [x] armv7l
- [x] i686

### CODE REVIEW

File: packages/autoconf213.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/autoconf213-2.13-chromeos-x86_64.tar.xz'
+  })
+  binary_sha256({
+    aarch64: '43464c65466ff94bb507a3174ddd0d325dd0e02d9ffe76a2575d44a795c8f699',
+     armv7l: '43464c65466ff94bb507a3174ddd0d325dd0e02d9ffe76a2575d44a795c8f699',
+       i686: 'c47998526c952a872745218099bf240c5e3105064608202cb8c2730186f4163b',
+     x86_64: '732cdb8cc6bd9be3847fca42713da472f83032a13322efb9eb19e4a23a9e9338'
+  })
+
+  depends_on 'perl'
```
Comment (uberhacker) [inline packages/autoconf213.rb:24]: Already a core package?

File: packages/autoconf213.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    system "env CFLAGS='-pipe -flto=auto' CXXFLAGS='-pipe -flto=auto' LDFLAGS='-flto=auto' \
+      ./configure #{CREW_OPTIONS} \
+      --infodir=#{CREW_PREFIX}/share/info \
+      --program-suffix=-2.13 \
+      -datadir=#{CREW_PREFIX}/share/autoconf213"
+    system 'make'
+  end
+
+  def self.install
+    system "make prefix=#{CREW_DEST_PREFIX} infodir=#{CREW_DEST_PREFIX}/share/info install"
```
Comment (supechicken) [inline packages/autoconf213.rb:34]: Does `DESTDIR` variable work?

**Comment by satmandu** [comment]
I don't believe it does. Hence using prefix again.

File: packages/autoconf213.rb (hunk 2 above)
Comment (satmandu) [reply packages/autoconf213.rb:34]: No, sadly, There's no DESTDIR in Makefile: https://github.com/autotools-mirror/autoconf/blob/autoconf-2-13-branch/Makefile.in

**Pull request merged**